audino/frontend/src/pages/annotate.js

```diff
@@ -227,15 +227,27 @@ class Annotate extends React.Component {
   }
 
   render() {
-    const { wavesurferMethods, maxHeight, disappear, referenceWindowOn, showActiveForm } =
+    let maxHeight;
+    try {
+      const leftWindow = document.getElementById("leftWindow")
+      console.log(leftWindow.scrollHeight)
+
+      maxHeight = Math.max(777, leftWindow.scrollHeight)+ "px"
+    } catch {
+      maxHeight = "100%"
+    }
+    
+    console.log(maxHeight)
+    const { wavesurferMethods, disappear, referenceWindowOn, showActiveForm } =
       this.state;
 
     if (wavesurferMethods) {
       wavesurferMethods.updateState(this.state);
     }
 
     return (
-      <div style={{ margin: 0, height: `${maxHeight}px`, overflow: 'hidden' }}>
+      <div style={{ margin: 0, height: maxHeight, overflow: 'hidden' }} //height: `${maxHeight}px`
```

Comment:
I'm still not able to scroll down all the way when I shrink my window height, but removing `overflow: hidden` should fix this.

Reply:
Resolved by latest push, be sure to test both reference window and without reference windows